_benchmark/workloads/corpora.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: corpora
+parent: Workload reference
+nav_order: 70
+---
+
+The `corpora` element contains all the document corpora used by the workload. You can use document corpora across workloads by copying and pasting any corpora definitions. 
+
+## Example
+
+The following example defines a single corpus called `movies` with `11658903` documents and `1544799789` uncompressed bytes.
+
+```json
+  "corpora": [
+    {
+      "name": "movies",
+      "documents": [
+        {
+          "source-file": "movies-documents.json",
+          "document-count": 11658903, # Fetch document count from command line
+          "uncompressed-bytes": 1544799789 # Fetch uncompressed bytes from command line
+        }
+      ]
+    }
+  ]
+```
+
+## Configuration options
+
+Use the following options with `corpora`.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+| `name` | Yes | String | The name the document corpus. Because OpenSearch Benchmark will use this name in its directories, use only lower case name without white spaces. |
```

Comment:
```suggestion
| `name` | Yes | String | The name the document corpus. Because OpenSearch Benchmark uses this name in its directories, use only lower case names without white spaces. |
```